recipe/recipe.yaml

```diff
@@ -0,0 +1,138 @@
+schema_version: 1
+
+context:
+  name: qe
+  version: 7.4
+  mpi: openmpi
+  mpi_prefix: "mpi_${{ mpi }}"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/QEF/q-e/archive/${{ name }}-${{ version }}.tar.gz
+  sha256: b15dcfe25f4fbf15ccd34c1194021e90996393478226e601d876f7dea481d104
+
+build:
+  number: 0
+  skip: win
+  script: build.sh
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - ${{ compiler('fortran') }}
+    - openmpi-mpifort
+    - cmake >=3.18.0
+    - make
+    - m4
+    - pkg-config
+    - wget
+    - if: linux
+      then: libgomp
+    - if: osx
+      then: llvm-openmp
+  host:
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+    - "hdf5 * *${{ mpi_prefix }}_*"
+    - openmpi
+    - scalapack
+    - "elpa * *${{ mpi_prefix }}_*"
+    - fftw
+    - fox
+    # libxc 5 conda-forge package currently does not include fortran bindings
+    # see https://github.com/conda-forge/libxc-feedstock/issues/15
+    #- libxc
+  run:
+    - fftw
+    - openmpi
+    - scalapack
+    - elpa
+
+tests:
+  - script:
```

Comment:
For file existence tests you can also use `package_contents`, e.g.:

```
- package_contents:
    bin:
      - alpha2f.x
      - bands.x
      ...      
```

Reply:
Thanks @wolfv. After change to using `packages_content` the parser seems not happy with it, any idea why? 

Check the error in https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1244529&view=logs&j=60238c3a-ba84-5ca5-39dc-717fbb421dbc&t=9cf66154-4ecc-5714-a3ee-b77d4233a250&l=1022

Reply:
Have to remove the requirements